src/pages/qna-notice/[id]/types.ts

```diff
@@ -13,6 +13,7 @@ export interface QnaDetail {
   department: string;
   college: string;
   qnaTargetCode: string;
+  authorname: string;
```

Comment:
```suggestion
  authorName: string;
```
단순 오타인 것 같아서 제안드리는데, 백엔드에서도 이런 필드 명이라면 놔두셔도 될 것 같아요.

Reply:
백엔드에서 기존 authorName과 다른 마스킹된 이름을 authortname 필드로 내려준다고 합니다.

Reply:
엥 그럼 authorName authorname 두개의 필드가 동시에 존재하는 상황인가요? 이름을 같게 하는게 베스트인 것 같고, 이름을 다르게 한다면  `authorNameMasked` 같은 정상적인 이름을 채택했으면 좋겠네요.

Reply:
의견 전달해 보겠습니다

Reply:
데이터 받아보니까 기존 authorName에 내려주네요! authorname은 type에서 제거했습니당